packages/tsconfig.options.json

```diff
@@ -11,12 +11,13 @@
     "moduleResolution": "node",
     "noImplicitReturns": true,
     "noImplicitThis": true,
+    "noImplicitAny": true,
```

Comment:
This should be enabled under strict, or is something else disabling it?

See https://www.typescriptlang.org/tsconfig#noImplicitAny

> Default:
true if strict,
false otherwise.

Reply:
Nice catch! Removed.